modules/updating-control-plane-only-update.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/control-plane-only-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="updating-control-plane-only-update_{context}"]
+= Control Plane Only update
+
+The following procedure pauses all non-master machine config pools and performs updates from {product-title} <4.y> to <4.y+1> to <4.y+2>, then unpauses the previously paused machine config pools.
+Following this procedure reduces the total update duration and the number of times worker nodes are restarted.
+
+.Prerequisites
+
+* Review the release notes for {product-title} <4.y+1> and <4.y+2>.
+* Review the release notes and product lifecycles for any layered products and Operator Lifecycle Manager (OLM) Operators. Some may require updates either before or during a Control Plane Only update.
```

Comment:
Per [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m) "may" should only be used to indicate permission, and "might" should be used to indicate possibility:
```suggestion
* Review the release notes and product lifecycles for any layered products and Operator Lifecycle Manager (OLM) Operators. Some might require updates either before or during a Control Plane Only update.
```